src/briefcase/integrations/cookiecutter.py

```diff
@@ -77,6 +77,28 @@ def escape_non_ascii(obj):
         environment.filters["escape_non_ascii"] = escape_non_ascii
 
 
+class GradleEscape(Extension):
+    """Jinja2 extension to escape strings for Gradle as well."""
+
+    def  __init__(self, environment):
+        """Initialize the extension with the given environment."""
+        super().__init__(environment)
+
+        def escape_toml(obj):
+            """Escapes single quotes."""
+            return obj.replace("'", '\\"')
```

Comment:
Is this escaping a single quote, or replacing a single quote with an escaped double quote?